shader/flatten.wgsl

```diff
@@ -76,41 +76,60 @@ struct EulerSeg {
     params: EulerParams,
 }
 
+// Threshold below which a derivative is considered too small.
```

Comment:
Somewhere in this file, I suggest adding a disclaimer comment that refers the reader to src/cpu_shaders/euler.rs and src/cpu_shaders/flatten.rs for more detailed comments.